### PR TITLE
fixes "Blob gas tastes of "slime and errors""

### LINKED ISSUE #26406: Blob gas tastes of "slime and errors"
@ChangelingRain assured me that blob chemicals would never be ingested. She was wrong.

### PR DESCRIPTION
**Pull request by ghost**
dunno what else to really do to fix this, made an attempt at an IC description.

fixes #26406

### CODE REVIEW

**Comment by Cyberboss** [comment]
If the base type is never used directly just make tastes for all the subtypes

File: code/modules/reagents/chemistry/reagents/blob_reagents.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	name = "Unknown"
 	description = "shouldn't exist and you should adminhelp immediately."
 	color = "#FFFFFF"
-	taste_description = "slime and errors"
+	taste_description = "gross mold"
```
Comment (Cyberboss) [inline code/modules/reagents/chemistry/reagents/blob_reagents.dm]: keep the slime and errors since it shouldnt show up

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — DISMISSED** [review]

File: code/modules/reagents/chemistry/reagents/blob_reagents.dm [hunk 2]
```diff
@@ -113,6 +115,7 @@
 	id = "shifting_fragments"
 	description = "will do medium brute damage."
 	effectdesc = "will also cause blob parts to shift away when attacked."
+	taste_description = "warped"
```
Comment (XDTM) [inline code/modules/reagents/chemistry/reagents/blob_reagents.dm]: Tastes like  warped?

File: code/modules/reagents/chemistry/reagents/blob_reagents.dm [hunk 3]
```diff
@@ -146,6 +149,7 @@
 	id = "blazing_oil"
 	description = "will do medium burn damage and set targets on fire."
 	effectdesc = "will also release bursts of flame when burnt, but takes damage from water."
+	taste_description = "hell"
```
Comment (XDTM) [inline code/modules/reagents/chemistry/reagents/blob_reagents.dm]: burning maybe? Or just oil

**Pull request merged**